packages/SqueakMaps-Core.package/SMAWindow.class/instance/searchLocationFromCoordinates..st

```diff
@@ -4,6 +4,13 @@ searchLocationFromCoordinates: aLatLong
 	| location zoom |
 	
 	zoom := self mapConfigurator getCurrentZoom.
-	location := self searchManager reverse: aLatLong withZoom: zoom.
+	[location := self searchManager reverse: aLatLong withZoom: zoom]
+		on: SMALocationUnnameableError 
+		do: [: error |
+			location := SMALocation new
```

Comment:
Wann genau tritt dieser Error auf? Habt ihr da vielleicht ein Beispiel?

Reply:
Das ist der Fehler der geworfen wird, wenn du beispielsweise auf das Meer oder das "Bir Tawil Triangle" in Afrika doppelklickst, um einen Pin zu setzen. Dafür kann dann kein Name gefunden werden. Der Fehler wird dann im Window abgefangen und anstatt den Fehler an den Nutzer weiterzugeben, wird ein Pin angezeigt. Dieser hat dann als Namen die Koordinaten der Location.

Reply:
okay super, danke!